modules/nwtc-library/src/SingPrec.f90

```diff
@@ -60,18 +57,18 @@ MODULE Precision
 INTEGER, PARAMETER              :: IntKi          = B4Ki                        !< Default kind for integers
 INTEGER, PARAMETER              :: BYTES_IN_INT   = 4                           !< Number of bytes per IntKi number    - use SIZEOF()
 
-#if !defined (DOUBLE_PRECISION) && !defined (OPENFAST_DOUBLE_PRECISION)
```

Comment:
Removing this `DOUBLE_PRECISION` preprocessor definition check made the vs-build project files not able to build in double precision any more. (Notice that the new code checks only for `OPENFAST_DOUBLE_PRECISION`). I am adding a fix for this in https://github.com/OpenFAST/openfast/pull/1008, but maybe we need to update documentation, too? See the original reason for the two preprocessor definitions here: https://github.com/OpenFAST/openfast/pull/524

Reply:
@deslaughter, can you take a look at this?  I think we need to fix it for 3.5.0

Reply:
@bjonkman, could I change the vs-build project files to define `OPENFAST_DOUBLE_PRECISION` instead of `DOUBLE_PRECISION`?

Reply:
@deslaughter You can cherrypick this commit with the changes to the vs-build files: https://github.com/OpenFAST/openfast/commit/9787e501d0fcf824b21f0b7c30d26057c4319804 Hopefully I got them all.

Reply:
@bjonkman, thanks for creating the commit. I've cherry-picked it into a bugfix branch and created PR #1560. I looked through the VS project files and updated `DOUBLE_PRECISION` in `AeroDyn_Inflow_c_binding.vfproj`.